src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1201,6 +1201,7 @@
     <Field Name="InvokedAsExtensionMethod" Type="bool"/>
     <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
     <Field Name="ResultKind" PropertyOverrides="true" Type="LookupResultKind"/>
+    <Field Name="BinderOpt" Type="Binder" Null="allow"/>
```

Comment:
><Field Name="BinderOpt" Type="Binder" Null="allow"/> [](start = 4, length = 52)

Please add a comment that this property is added as a temporary solution for IOperation implementation and should probably be removed in the future. Same suggestion for other instances as well.